straxen/plugins/acqmon_processing.py

```diff
@@ -216,3 +216,125 @@ def compute(self, events, aqmon_hits):
         result['time'] = events['time']
         result['endtime'] = events['endtime']
         return result
+
+
+DIGITIZER_DELAY_CHANGE_RUNID = '021286'
+TO_BE_EXCPECTED_MAX_DELAY = 11e3
+
+@strax.takes_config(
+    strax.Option('tpc_internal_delay', default=[4817, 10137], type=int, track=True,
```

Comment:
probably worth making an immutabledict?

Reply:
just to remove the hardcoding required

Reply:
also worth making a straxen config instead of an option so that we can change it easily to a CMT option if we ever wanted to

Reply:
Unfortunately immutabledicts are not json serializable for what ever reason.  So we can either untrack the option or have to use something different. Maybe not tracking it is fine. What do you think?